app/src/main/sentences/ru/current_time.dslf

```diff
@@ -0,0 +1,2 @@
+current_time: high
+(сколько сейчас? времени)|(который час)?;
```

Comment:
With the `?` the sentence could unfold to an empty sentence, which would match anything. Did you mean something in particular by adding that `?`? Anyway, for now I am removing it and merging, in case just write a comment here ;-)
```suggestion
(сколько сейчас? времени)|(который час);
```